vendor/wp-now/src/wp-now.ts

```diff
@@ -135,7 +135,7 @@ export default async function startWPNow(
 			await prepareWordPress( php, options );
 			return runtimeId;
 		},
-		maxRequests: 400,
+		maxRequests: 10,
```

Comment:
Is this change here only for test purposes?

Reply:
yeah, this shouldn't go here 😅 . Good catch.